builds/e2e/connectivity.yaml

```diff
@@ -286,7 +286,7 @@ jobs:
         - agent-osbits -equals 32
         - run-connectivity -equals true
     variables:
-      edgelet.artifact.name: 'iotedged-debian9-arm32v7'
+      edgelet.artifact.name: 'iotedged-debian10-arm32v7'
```

Comment:
For all of the test pipelines here, you should change it to the _latest_ version of debian, so 11 instead of 10.